src/documents/WebGamesAndSimulations.md

```diff
@@ -0,0 +1,164 @@
+* Content contained in the background (using background CSS property), :before or :after elements are ignored by screen readers. Do not put important content within these elements. Use standard elements like `<image>`, `<div>`, `<figure>`, etc. as needed.
+* Images which are in the background should be cosmetic and not critical to the outcomes. Use `<img>` and `<figure>` as necessary for important graphics / images.
+* Ensure good contrast in colours and details. You can check your colours by using the <a href="http://webaim.org/resources/contrastchecker/" rel="nofollow" target="_blank" class="link-external">WebAIM Color Contrast Checker</a>.
+
+## Punctuation
+
+Punctuation helps a screen reader know when to pause, or give proper emphasis.
+
+* Check for missing punctuation - if periods or commas are missing, screen readers would not know to pause.
+* Unicode characters are often omitted by screen readers. Ellipsis (...), arrows, and other characters are ignored.
+* Acronyms and abbreviations should use the `<abbr>` element with the relevant aria properties. For example:
+    * `<abbr role="text" title="kilograms" aria-label="kilograms">kg</abbr>`
+    * `<abbr role="text" title="miles per hour" aria-label="miles per hour">mph</abbr>`
+
+##  Charts and Graphs
+
+Charts and graphs are an excellent way to convey data in a meaningful way. However, charts and graphs typically favour a student who is capable of perceiving the content and can cognitively process the information.
+
+Some methods in providing a more inclusive experience of charts and graphs:
+
+* Provide alternatives to charts. Give text descriptions and interpretations of the data, give a table of data.
+* A good description identifies:
+    * What is being shown (i.e. the purpose of the chart)
+    * Describe the values (relative sizes and differences between values).
+    * Describe any patterns to may help us understand the data.
+    * Summarize the trend / data pattern.
+* Also see:
+    * PSU’s chart accessibility guide - <a href="http://accessibility.psu.edu/images/charts/" rel="nofollow" target="_blank" class="link-external">http://accessibility.psu.edu/images/charts/</a>
+    * W3C's guide on describing complex images - <a href="http://www.w3.org/WAI/tutorials/images/complex/" rel="nofollow" target="_blank" class="link-external">http://www.w3.org/WAI/tutorials/images/complex/</a>
+
+If using a table of data:
+
+* Follow best practices for table formatting (i.e. proper use of table headers, no table nesting). <a href="https://www.webaccessibility.com/best_practices.php?technology_platform_id=8" rel="nofollow" target="_blank" class="link-external">See Data Table Best Practices at WebAccessibility.com</a>
+* Any math, symbols, units, or acronyms defined. For additional information about making math accessible, see [Mathematics](Mathematics.html).
```

Comment:
Change the form of this to the active "Define any math, symbols..." Perhaps same for next point? Put the "Simplify..." first?